course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -624,25 +624,15 @@ describe('Unit testing PeopleController', function() {
             expect(result).toEqual('Manually Added');
         });
 
-        it('renderRemove for registrar-fed', function() {
-            var full = {source: 'fasfeed', user_id: '1234567890'};
-            var meta = {row: 1};
-            var result = scope.renderRemove(undefined, undefined, full, meta);
-            var expectedResult = '<div class="text-center">' +
-                                 '<i class="fa fa-trash-o fa-trash-disabled">' +
-                                 '</i></div>';
-            expect(result).toEqual(expectedResult);
-        });
-
-        it('renderRemove for manual', function() {
+        it('renderRemove should render correctly', function() {
             var full = {source: 'peopletool', user_id: '9876543210'};
```

Comment:
Maybe we could check that `{source: 'fasfeed', user_id: '1234567890'}` also renders as expected, as a regression check.